templates/includes/footer.html

```diff
@@ -1,2 +1,79 @@
+                </li>
+                <li>
+                    <!-- To link to own Plagiarism Policy page once built -->
+                    <a href="{% url 'plagiarism_policy' %}" target="_blank">Plagiarism Policy</a>
+                </li>
+                <li>
+                    <a href="https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-terms-of-service"
+                        target="_blank">Github Terms of Service</a>
+                </li>
+            </ul>
+        </div>
+        <div class="col-sm-6 col-md-3">
+            <h4>
+                Resources
+            </h4>
+            <ul>
+                <li>
+                    <!-- to link to FAQ page once built -->
+                    <a href="{% url 'faq' %}" target="_blank">FAQ</a>
+                </li>
+                <li>
+                    <!-- to link to Judging Criteria page once built -->
+                    <a href="{% url 'judging_criteria' %}" target="_blank">Judging Criteria</a>
+                </li>
+                <li>
+                    <!-- to link to Useful Resources page once built -->
+                    <a href="{% url 'useful_resources' %}" target="_blank">Useful Resources</a>
+                </li>
+                <li>
+                    <a href="https://codeinstitute.net/" target=" _blank">Code Institute</a>
+                </li>
+                <li>
+                    <a href="https://hacktoberfest.digitalocean.com/" target="_blank">Hacktoberfest Official Site</a>
+                </li>
+                
+            </ul>
+        </div>
+        <div class="col-md-6">
+            <h4>
+                About Us
+            </h4>
+            <p>
+                Code Institute is the leading global provider
+                of online coding education. Credit-rated by Edinburgh Napier University and validated by the Higher
+                Education Authority in Sweden, the Ministry of Communications in Singapore, and the Higher Education
+                Authority in Ireland.
+            </p>
+        </div>
+        <hr>
+        <div class="col github-wrapper">
+            <a href="https://www.facebook.com/Code.InstituteIRL/" target="_blank">
+                <i class="fab fa-facebook-square"></i>
+            </a>
+            <a href="https://twitter.com/codersinstitute" target="_blank">
+                <i class="fab fa-twitter-square"></i>
+            </a>
+            <a href="https://www.linkedin.com/school/code-institute/" target="_blank">
+                <i class="fab fa-linkedin"></i>
+            </a>
+            <a href="https://github.com/Code-Institute-Community/ci-hackathon-app" target="_blank">
+                <i class="fab fa-github"></i>
+            </a>
+        </div>
+    </div>
+    <div class="col copyright-wrapper">
+        <p>
+            <a href="https://codeinstitute.net/" target=" _blank">&copy; Copyright 2020 Code Institute</a>

```

Comment:
@JimLynx 
`&copy;` with **Copyright** is redundant, let's remove the word 'Copyright'.